src/lammpsio/topology.py

```diff
@@ -0,0 +1,189 @@
+import numpy
+
+
+class Topology:
+    """Topology of configuration.
+
+    Parameters
+    ----------
+    N : int
+        Number of connections.
+    num_members : int
+        Number of members in a connection.
```

Comment:
Can you clarify what you mean here?

Reply:
`num_types` is documented as an attribute of the class (i.e., it can be accessed like `Topology.num_types`), but not as the third argument of the constructor. It needs to be in both places.